collect_app/src/androidTest/java/org/odk/collect/android/AllWidgetsFormTest.java

```diff
@@ -841,8 +843,20 @@ public void testLabelWidget() {
 
     public void testTriggerWidget() {
 
+        boolean checkBoxBoolean = randomBoolean();
```

Comment:
Boolean might be `true` or `false` (two cases) so I think you should just test both. It doesn't make sense to use `randomBoolean()` here I think.